CONTRIBUTING.md

```diff
@@ -86,6 +91,24 @@ As a contributor, you are expected to fork this repository, work on your own for
 
     *IMPORTANT* Please pay attention to the maintainer's feedback, since its a necessary step to keep up with the standards OpenZeppelin Contracts attains to.
 
+## Documentation
+
+Before submitting the PR, you must update the corresponding documentation entries in the docs folder. In the future we may use something similar to solidity-docgen to automatically generate docs, but for now we are updating .adoc entries manually.
+
+If you want to run the documentation UI locally:
+
+1. Change directory into docs inside the project and run npm install.
+
+    ```bash
+    cd docs && npm i
+    ```
+
+2. Build de docs and run the local server (default to localhost:8080). This will watch for changes in the docs/module folder, and update the UI accordingly.
+
+    ```bash
+    npm run docs:watch
```

Comment:
after following these steps I get these errors:

```

> docs@0.0.1 docs:watch /home/cairo-contracts/docs
> npm run docs watch ../src


> docs@0.0.1 docs /home/cairo-contracts/docs
> node_modules/@openzeppelin/docs-utils/oz-docs.js -c . "watch" "../src"

Cloning into '/root/.cache/openzeppelin-docs-preview/cef0ec7d377ce120ebe08cb50dbf37bb6ebe12bd'...
remote: Enumerating objects: 221, done.
remote: Counting objects: 100% (221/221), done.
remote: Compressing objects: 100% (191/191), done.
remote: Total 221 (delta 15), reused 154 (delta 12), pack-reused 0
Receiving objects: 100% (221/221), 1.51 MiB | 744.00 KiB/s, done.
Resolving deltas: 100% (15/15), done.
00h00m00s 0/0: : ERROR: There are no scenarios; must have at least one.
Detected source changes, rebuilding docs...
npm ERR! path /home/cairo-contracts/src/openzeppelin/token/erc721/enumerable/presets/utils/package.json
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open '/home/cairo-contracts/src/openzeppelin/token/erc721/enumerable/presets/utils/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-09-07T17_56_37_982Z-debug.log
Detected docs changes, rebuilding site...

> docs.openzeppelin.com@0.0.0 build:custom /root/.cache/openzeppelin-docs-preview/cef0ec7d377ce120ebe08cb50dbf37bb6ebe12bd
> bash scripts/build.sh "/root/.cache/openzeppelin-docs-preview/cef0ec7d377ce120ebe08cb50dbf37bb6ebe12bd/local-playbook.yml"

+ '[' -n '' ']'
+ antora --stacktrace /root/.cache/openzeppelin-docs-preview/cef0ec7d377ce120ebe08cb50dbf37bb6ebe12bd/local-playbook.yml
scripts/build.sh: line 26: antora: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! docs.openzeppelin.com@0.0.0 build:custom: `bash scripts/build.sh "/root/.cache/openzeppelin-docs-preview/cef0ec7d377ce120ebe08cb50dbf37bb6ebe12bd/local-playbook.yml"`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the docs.openzeppelin.com@0.0.0 build:custom script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-09-07T17_56_38_166Z-debug.log
The site is available at http://localhost:8080
```

and although it spins up the server, it doesn't work:

<img width="416" alt="image" src="https://user-images.githubusercontent.com/1703831/188949582-589b8108-2146-4cad-aa09-4e6584c31718.png">

manually installing `antora` in the `docs/` folder produces the following:

```
+ '[' -n '' ']'
+ antora --stacktrace /root/.cache/openzeppelin-docs-preview/cef0ec7d377ce120ebe08cb50dbf37bb6ebe12bd/local-playbook.yml
[15:26:02.812] FATAL (antora): Cannot find module 'lunr'
Require stack:
- /root/.cache/openzeppelin-docs-preview/cef0ec7d377ce120ebe08cb50dbf37bb6ebe12bd/lunr-tokenizer.js
- /home/cairo-contracts/docs/node_modules/@antora/user-require-helper/lib/index.js
- /home/cairo-contracts/docs/node_modules/antora/node_modules/@antora/cli/lib/cli.js
- /home/cairo-contracts/docs/node_modules/antora/node_modules/@antora/cli/lib/index.js
- /home/cairo-contracts/docs/node_modules/antora/bin/antora
```
`npm i lunr` doesn't fix it. not sure if it's a problem on my end or not

Reply:
Weird. You shouldn't have to install antora manually (second error). I removed the first error with the last commit.

Reply:
It looks like what is failing is the cloning of the @openzeppelin/docs-utils repo (step from oz-docs script). Maybe the script didn't receive the permission to clone this?